be/src/exprs/bloomfilter_predicate.h

```diff
@@ -65,6 +65,10 @@ class BlockBloomFilterAdaptor {
         return _bloom_filter->find(Slice(data, len));
     }
 
+    bool test_uint32_t(uint32_t data) const {
```

Comment:
should refactor code of `test_bytes` and `test_uint32_t` use template to make code clear.